libraries/ae-inventory-processor/src/main/java/org/metaeffekt/core/inventory/processor/report/model/aeaa/AeaaVulnerabilityContextInventory.java

```diff
@@ -431,6 +432,8 @@ protected void parse(List<Artifact> artifacts, List<VulnerabilityMetaData> vmds,
                         final List<Artifact> artifactsForVulnerability = artifactEntry.getValue();
                         for (Artifact artifact : artifactsForVulnerability) {
                             vulnerability.manuallyAffectsArtifact(key, artifact);
+                            vulnerability.manuallyAffectsAssets(InventoryUtils.getAssetsForArtifact(inventory,
```

Comment:
- If you change something in a class prefixed with `Aeaa`, you will have to find the corresponding file in the [Artifact Analysis project](https://github.com/org-metaeffekt/metaeffekt-artifact-analysis/blob/429b60a1103db79ad083db179561d7d19e689107/modules/ae-artifact-analysis/src/main/java/com/metaeffekt/mirror/contents/base/VulnerabilityContextInventory.java#L44) to update as well.
- The same goes for the `Vulnerability` and it's superclasses.